src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -70,6 +70,7 @@ public function iAmLoggedInAs($email)
 
     /**
      * @Given /^I am logged in customer$/
+     * @Given /^I am logged in as customer$/
```

Comment:
Yes, because it should be change to "as customer" but I want to do this i other PR.